grouper/settings.py

```diff
@@ -32,16 +39,31 @@ def update_from_config(self, filename, section=None):
             if override is not None and callable(override):
                 value = override(value)
 
+            self[key] = value
+
+
+    def update_from_config(self, filename, section=None, refresh_config_seconds=10):
+        self._update_from_config(filename, section=section)
+        def refresh_config_loop():
+            while True:
+                time.sleep(refresh_config_seconds)
+                self._update_from_config(filename, section=section)
```

Comment:
Yup!  Really we should expose the stats via the /debug/stats route like the other asynchronous methods.